src/Gui/NavigationStyle.cpp

```diff
@@ -1020,8 +1020,7 @@ void NavigationStyle::moveCursorPosition()
 
 void NavigationStyle::redraw()
 {
-    if (mouseSelection)
-        mouseSelection->redraw();
```

Comment:
What's going on with this removal?

Reply:
With AbstractMouseSelection::redraw() also NavigationStyle::redraw() can be completely removed.